tests/checkpointing/test_model_checkpoint.py

```diff
@@ -625,7 +608,6 @@ def test_ckpt_every_n_train_steps(tmpdir):
     epoch_length = 64
     checkpoint_callback = ModelCheckpoint(
         filename="{step}",
-        every_n_epochs=0,
```

Comment:
The triggers aren't exclusive now so setting it to 0 would mean we save nothing.